README.md

```diff
@@ -1670,6 +1670,11 @@ The following extractors use this feature:
 * `language`: Languages to extract. Eg: `crunchyroll:language=jaJp`
 * `hardsub`: Which hard-sub versions to extract. Eg: `crunchyroll:hardsub=None,enUS`
 
+#### crunchyroll:beta
+* `format`: Which stream type(s) to extract. Default is `adaptive_hls` Eg: `crunchyrollbeta:format=vo_adaptive_hls`
+    * Potentially useful values include `adaptive_hls`, `adaptive_dash`, `vo_adaptive_hls`, `vo_adaptive_dash`, `download_hls`, `trailer_hls`, `trailer_dash`
+* `hardsub`: Preference order for which hardsub versions to extract. Default is `None` (no hardsubs). Eg: `crunchyrollbeta:hardsub=en-US,None`
```

Comment:
Both options should default to extracting all formats/subs. It is more important to make all data available by default than to reduce a few network requests. The extractor-arg can be used to limit it

Reply:
As a user, I'd much prefer the hardsubs not show up normally, since they're basically never what you want, and it's a waste to go fetch them all.

And do you really want the default form of the extractor to make 45 requests? Serially? Taking 9 seconds in my example test? And end up with 400 formats? Many of which are precisely identical data just coming from different servers?

I mean, you're the boss, but I don't think this is a good idea. At the very least, it should probably advertise the extractor args clearly in the output, because otherwise a lot of people will just sit through this, not realizing it could be way faster. Indirect users, like those using it through `mpv`s hook script, won't even see that though :-/